client/src/cb-admin/pages/Settings.js

```diff
@@ -94,6 +101,10 @@ export default class SettingsPage extends React.Component {
       errors: {},
       form: {},
       dropzoneMsg: undefined,
+      users: [],
+      visits: [],
+      visitsString: 'This is visitsString!',
```

Comment:
What are these?

Reply:
users and visits are the data I'm getting back from the db; and the strings are the csv strings generated by fast-csv which jszip uses to generate the files within the zip.

Reply:
Yeah, but why is there irrelevant default data? Empty strings are more appropriate.